test/unit-tests/ctest.test.ts

```diff
@@ -0,0 +1,24 @@
+import { readTestResultsFile } from "@cmt/ctest";
+import { expect, getTestResourceFilePath } from "@test/util";
+
+suite('CTest test', () => {
```

Comment:
is this test created to test the function "readTestResultsFile"  only ? The XML files are not being generated each time as a result of another test, right?

Reply:
Eventually we should add coverage for actual running of ctest.  I just wanted a baseline for the parsing for now because it was broken earlier.  We could replace the checked in xml file with one that is produced by ctest.